### DISCUSSION TITLE
TuringCourses/major_elective/computational_photography/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#65] [General]
# TuringCourses/major_elective/computational_photography/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major_elective/computational_photography/

**Comment by ZhouTimeMachine** [comment]
我可能会担任 2024 春夏计算摄影学的助教，届时若有空可能会修改实验

**Comment by geohaolg** [comment]
课件好像在网站上失效了
